text/0000-partial-template.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Partial Template
+status: DRAFTED
+created_at: 2020-01-16
+updated_at: 2020-01-17
+pr: (leave this empty until the PR is created)
+---
+
+# Short Title
+
+## Summary
+
+Partial templates allow the ability to import html templates into another template via
+the normal variable syntax `{template}`.
+
+## Basic example
+
+```js
+import partialTemplate from './partialTemplate.html';
+
+export default class Example extends LightningElement {
+    partialTemplate = partialTemplate;
+}
+```
+
+```html
+<template>
+  Parent
+  {partialTemplate}
+</template>
+```
+
+## Motivation
+
+In investigating performance issues in components that are used hundreds of times on a

```

Comment:
I don't think we use perf as one of the motivations because there are multiple ways to improve the performance of this. cc @pmdartus

Reply:
@caridy I'll rewrite this to explain it in another way. This quip/demo outlines how I was researching ideas: https://salesforce.quip.com/9KGnAAWaHRNr

Reply:
this is a public repo @Templarian, don't share internal links.

Reply:
Agreed with that reasoning. Resolving this as the main motivation is organization/reuse of templates. Perf is a "byproduct".